### PR TITLE
Add file structure explanation to README for contributions

### PR DESCRIPTION
**Pull request by gadhagod**
@goombamaui please let me know if I missed anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+#### Extension files
+These files are directly part of the installed chrome extension.
+* `pdfview.js` changes `/course/*/materials/gp/*` to use chrome's PDF viewer.
+* `pdfHeaderRemover.js` changes Schoology-hosted PDF's headers and mimetypes.
+* `manifest.json` contains the configuration, permissions, and scripts of the extension.
+* `main.js` applies to all schoology routes.
+* `lunch.js` retrieves and displayes the lunch PDF at `/lunch`.
+* `jQuery.js` is the [jquery library](https://jquery.com).
+* `chart.js` is the [chart.js library](https://jquery.com/).
+* `background.js` is the background script of this extenison.
```
Comment (gadhagod) [inline README.md:16]: @goombamaui can this be elaborated on?

**Review by goombamaui — CHANGES_REQUESTED** [review]
chart.js has the link to jquery.com, is this intentional

**Comment by gadhagod** [comment]
> chart.js has the link to jquery.com, is this intentional

Fixed now.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+These files are directly part of the installed chrome extension.
+
+-   `pdfview.js` changes `/course/*/materials/gp/*` to use chrome's PDF viewer.
+-   `pdfHeaderRemover.js` changes Schoology-hosted PDF's headers and mimetypes.
+-   `manifest.json` contains the configuration, permissions, and scripts of the extension.
+-   `main.js` applies to all schoology routes.
+-   `lunch.js` retrieves and displayes the lunch PDF at `/lunch`.
+-   `jQuery.js` is the [jquery library](https://jquery.com).
+-   `chart.js` is the [chart.js library](https://www.chartjs.org).
+-   `background.js` is the background script of this extenison.
```
Comment (gadhagod) [inline README.md:18]: > -   `background.js` is the background script of this extenison.

@goombamaui, should this be elaborated on?

**Review by goombamaui — APPROVED** [review]

**Pull request merged**